<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>GeometryFactory</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MWVD</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout">
    <item row="1" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditTime">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Step size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditStepSize"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrevEv"/>
   <addaction name="actionDecrTime"/>
   <addaction name="actionIncrTime"/>
   <addaction name="actionNextEv"/>
   <addaction name="separator"/>
   <addaction name="actionToggleVorDiag"/>
  </widget>
  <action name="actionIncrTime">
   <property name="text">
    <string></string>
   </property>
   <property name="toolTip">
    <string>Increase time</string>
   </property>
   <property name="font">
    <font>
     <family>FontAwesome</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionDecrTime">
   <property name="text">
    <string></string>
   </property>
   <property name="toolTip">
    <string>Decrease time</string>
   </property>
   <property name="font">
    <font>
     <family>FontAwesome</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionNextEv">
   <property name="text">
    <string></string>
   </property>
   <property name="toolTip">
    <string>Next event</string>
   </property>
   <property name="font">
    <font>
     <family>FontAwesome</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionPrevEv">
   <property name="text">
    <string></string>
   </property>
   <property name="toolTip">
    <string>Previous event</string>
   </property>
   <property name="font">
    <font>
     <family>FontAwesome</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionToggleVorDiag">
   <property name="text">
    <string></string>
   </property>
   <property name="toolTip">
    <string>Toggle Voronoi diagram</string>
   </property>
   <property name="font">
    <font>
     <family>FontAwesome</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
